hamilton/driver.py

```diff
@@ -472,8 +546,11 @@ def raw_execute(
         overrides: Dict[str, Any] = None,
         display_graph: bool = False,
         inputs: Dict[str, Any] = None,
+        _fn_graph: graph.FunctionGraph = None,
     ) -> Dict[str, Any]:
-        """Raw execute function that does the meat of execute.
+        """Raw execute function that does the meat of execute. This entrypoint for running the graph
+        will be deprecated in favor of using Base.DictResult(), which will be the default,
+        and already forms the default in the new builder pattern.
```

Comment:
Calling `.raw_execute()`. Rather, you call:

```python
dr = driver.Builder()...
dr.execute(...)
```
Or explicitly pass in the `base.DictResult()` result builder

Reply:
Right -- so the comment should direct users to what they should be doing. i.e. 
```suggestion
        """Raw execute function that does the meat of execute. 
        
        Don't use this entry point for execution directly. Always go through `.execute()`.
        In case you are using `.raw_execute()` directly, please switch to `.execute()` using a
        `base.DictResult()`. Note: `base.DictResult()` is the default return of execute if you are
        using the `driver.Builder()` class to create a `Driver()` object.
```
Otherwise I'm not worried, we didn't publicize using `raw_execute` much.